libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/camera/SimpleCamera.kt

```diff
@@ -0,0 +1,63 @@
+package com.mapbox.services.android.navigation.v5.navigation.camera
+
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.core.constants.Constants
+import com.mapbox.geojson.LineString
+import com.mapbox.geojson.Point
+import java.util.ArrayList
+
+/**
+ * The default camera used by [com.mapbox.services.android.navigation.v5.navigation.MapboxNavigation].
+ *
+ * @since 0.10.0
+ */
+open class SimpleCamera : Camera() {
+
+    companion object {
```

Comment:
have we aligned on declaring companion objects in the beginning of the class? if so, then should we move comp object in `RouteInformation.kt` to the top?